config/initializers/sidekiq.rb

```diff
@@ -35,4 +35,5 @@
   config.reaper_count    = 1000
   config.reaper_interval = 600
   config.reaper_timeout  = 150
+  config.lock_ttl        = 50.days.to_i
```

Comment:
My only concern is that potentially causing issues if we ever bump the number of retries or change the retry function.